static/cart.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="index.css">
+  <title>Document</title>
+</head>
+<body>
+  <div class="app layout">
+    <div class="layout__header header">
+      <div class="header__cart-info">
+        <button class="cart-info-cta__button"><</button>
+        <h1 class="cart-info__title">Tu Carrito (4)</h1>
+      </div>
+      <div class="header__subtotal">
+        <h2>Subtotal<br>
+        43,00 €</h2>
+        <button class="subtotal-cta__button">Tramitar Pedido</button>
+      </div>
```

Comment:
Same as the previous comment, let's review step by step this molecule too.
<img src="https://user-images.githubusercontent.com/17853818/104795132-5a8b4480-57ac-11eb-8aa2-dfc308416d87.png" width="320">

1. Name this UI component. Example: `subtotal-cta`
2. Think about its anatomy. Subtotal CTA molecule is made of the following atoms:
   * A title (with font-size 24, same as button and total title at bottom of page)
   * A button (same as used at bottom of the page)
<img src="https://user-images.githubusercontent.com/17853818/104795360-fcf7f780-57ad-11eb-8464-a0ed9e0c9a5f.png" width="320">

3. We apply the class names:
```html
      <div class="subtotal-cta">
        <h2 class="subtotal-cta__text title-x">Subtotal<br>
        43,00 €</h2>
        <button class="subtotal-cta__button button">Tramitar Pedido</button>
      </div>
```

4. We separate attributes in their correspondent classes:
```css
/** Atoms **/
.title-x {
  font-size: 24px;
}

.button {
  font-size: 24px;
  background-color: lightblue;
  display: inline;
  width: 100%; // by default buttons will be as wide as possible
}

/** Molecules **/
.subtotal-cta {
  display: flex;
  justify-content: space-evenly;
  border: 1px solid black;
  padding: 10px;
  margin: 10px;
}

.subtotal-cta__text {
  width: 100%;
}

.subtotal-cta__button {
  max-width: 180px; // width is overwritten for buttons inside subtotal cta molecule
  min-width: 80px;
}
```